src/k8s/deployment.ts

```diff
@@ -17,6 +17,10 @@ export class Command {
         return `oc get deploymentConfig -o json`;
     }
 
+    static showLogDc(deploymentConfig: string) {
```

Comment:
`showDeploymentConfigLog` would be better, because other methods are using full names as well.